resources: patch 7.0 data updates

wexxlee opened the pull request:
**NOTE: File updates contain 7.0 spoilers.  Review at your own peril.**

Old XIVAPI was updated before the beta, which is pending updates to EXDSchema, so this PR was generated based on the old api.  

There were schema & data changes that required code fixes beyond just the data refresh, so I think it makes sense to merge this first, and then fix and re-test the beta API refactor before merging #171.

MaikoTan reviewed (approved):
Overall looks good to me, just curious why are there `(6.3)` or `(6.4)` prefixes.

wexxlee commented:
> Overall looks good to me, just curious why are there `(6.3)` or `(6.4)` prefixes.

It seems like when the synthetic zone info was added for those deprecated zones for patch 6.4 and 6.5, the prefixes were omitted so they never made it to zone_info.  I added them to the synthetic zone data when I converted the script from py to ts a few months ago, but there was a small logic bug, wherein the synthetic zone names were not being reflected correctly in the output file.  I just happened to catch it and fix it with this update.
